examples/vgg/README.md

```diff
@@ -0,0 +1,13 @@
+# VGG
+
+For evaluation, please go to [`examples/classification`](https://github.com/chainer/chainercv/tree/master/examples).
+
+## Convert Caffe model
+Convert `*.caffemodel` to `*.npz`.
+
+```
+$ python caff2npz_vgg_16.py <source>.caffemodel <target>.npz
```

Comment:
Sorry, I forgot to change this line. Please update `caff2npz_vgg_16.py` -> `caffe2npz.py`.